.github/workflows/designer-dotnet-test.yaml

```diff
@@ -37,6 +37,6 @@ jobs:
       - name: Test
         run: |
            dotnet test backend/Designer.sln --filter Category!=GiteaIntegrationTest -v m
-      - name: FormatCheck
-        run: |
-           dotnet format backend/Designer.sln --verify-no-changes
+      # - name: FormatCheck
```

Comment:
Why are we skipping format check?

Reply:
Currently new dotnet sdk have issues with dotnet format command. It fails on github runners. I think the only thing we can do is to wait for it to be resolved and uncomment it when is fixed.